app/javascript/components/mappers/soqlToDomain/lease.js

```diff
@@ -0,0 +1,17 @@
+export const mapLease = (value) => {
```

Comment:
I thought that we had killed this function a couple of months ago. I believe all of the lease domain date-updating should be happening on the backend now

Reply:
hmmm, I'll try, but can we map date array to an json object in the backend?

Reply:
Here's the code where we're doing the conversion: https://github.com/Exygy/sf-dahlia-lap/blob/116d1351172dc1057acc68f2989a22c92615ae58/app/models/force/lease.rb#L25

it was already working so not sure why it's not working now